src/Oscar.jl

```diff
@@ -402,40 +402,56 @@ end
 
 
 @doc Markdown.doc"""
-    test_module(x, new::Bool = true)
+    test_module(;file="runtests", new::Bool = true)
 
-Run the Oscar tests in the file `x.jl` where `x` is a string.
-
-If `x == "all"` run the entire testsuite.
+Run the Oscar tests in the file `test/<file>.jl` where `file` may be a path.
+The default is to run the entire test suite.
 
 The optional parameter `new` takes the values `false` and `true` (default). If
 `true`, then the tests are run in a new session, otherwise the currently active
 session is used.
+
+For experimental modules, use [`test_experimental_module`](@ref) instead.
 """
-function test_module(x::AbstractString, new::Bool = true)
-   julia_exe = Base.julia_cmd()
-   if x == "all"
-     test_file = joinpath(oscardir, "test/runtests.jl")
-   else
-     test_file = joinpath(oscardir, "test/$x.jl")
-   end
+function test_module(;file::AbstractString="runtests", new::Bool=true)
```

Comment:
For `test_module` I'd make the `file` argument mandatory, since for the entire testsuite one could just use
```
]test Oscar
```
Sorry if this was unclear. Do you think that is too inconsistent?

Reply:
Hmm, I see your point. So maybe make it mandatory, but still a kwarg?

Reply:
I thought plain, like `project` for the `test_experimental_module`.